src/compiler/sys.ts

```diff
@@ -1,3 +1,4 @@
+import * as ts from "./_namespaces/ts";

```

Comment:
Is this being used, or was this a merge conflict problem?

Reply:
this is used and is to pass `ts` to plugin factory

Reply:
Ah, yep, missed it below.